consensus/hotstuff/forks/forks.go

```diff
@@ -40,13 +40,13 @@ type Forks struct {
 	forest   forest.LevelledForest
 
 	finalizationCallback module.Finalizer
-	newestView           uint64   // newestView is the highest view of block proposal stored in Forks
-	lastFinalized        *BlockQC // lastFinalized is the QC that POINTS TO the most recently finalized locked block
+	newestView           uint64                // newestView is the highest view of block proposal stored in Forks
```

Comment:
strictly speaking we don't need to maintain `newestView` anymore

Reply:
👍 will clean that up as part of my next PR extending forks